tests/cpu_bugs/l1tf.pm

```diff
@@ -51,13 +51,44 @@ my %mitigations_list =
 
 
 sub run {
+    my $self = shift;
     if (check_var('BACKEND', 'qemu')) {
         record_info('softfail', "QEMU needn't run this testcase");
         return;
     }
     my $obj = Mitigation->new(\%mitigations_list);
     #run base function testing
     $obj->do_test();
+
+    assert_script_run('lscpu | grep "0-19"');
+    assert_script_run('cat /sys/devices/system/cpu/smt/active | grep "1"');
+    assert_script_run('echo off>/sys/devices/system/cpu/smt/control');
+    assert_script_run('lscpu | grep "Off-line CPU(s) list"');
+    assert_script_run('cat /sys/devices/system/cpu/smt/active | grep "0"');
+    add_grub_cmdline_settings("l1tf=full,force");
+    grub_mkconfig;
+    Mitigation::reboot_and_wait($self, 150);
+    assert_script_run('lscpu | grep "Off-line CPU(s) list"');
+    assert_script_run('cat /sys/devices/system/cpu/smt/active | grep "0"');
+    assert_script_run('cat /sys/devices/system/cpu/smt/control | grep "forceoff"');
+    my $ret = script_run('echo on>/sys/devices/system/cpu/smt/control');
+    if ($ret eq 0) {
+        record_info('fail', "Control cannot be modified under the mode of forceoff");
+        die;
+    }
+    assert_script_run('cat /sys/devices/system/cpu/smt/control | grep "forceoff"');
+    remove_grub_cmdline_settings("l1tf=full,force");
+    grub_mkconfig;
+    Mitigation::reboot_and_wait($self, 150);
+    assert_script_run('cat /sys/module/kvm_intel/parameters/ept | grep "Y"');
+    my $damn = script_run('modprobe -r kvm_intel | grep "kvm"');
+    if ($damn eq 0) {
+        record_info('fail', "Couldn't find kvm when removed the kvm_intel");
+        die;
+    }
+    assert_script_run('modprobe kvm_intel ept=0;lsmod | grep "kvm"');
+    assert_script_run('cat /sys/module/kvm_intel/parameters/ept | grep "N"');
```

Comment:
again just an advice to increase readability of code : 
```suggestion
    check_param('/sys/module/kvm_intel/parameters/ept',"N"');    
```

where `check_param` : 
```
sub check_param {
   my ($param,$value)= @_;
    assert_script_run("cat $param | grep $value");
}
```

Reply:
Thanks, already changed